server/doc.js

```diff
@@ -0,0 +1,37 @@
+const mongoose = require('mongoose');
+mongoose.promise = Promise;
+const Schema = mongoose.Schema;
+const date = require('date-and-time');
+
+const docSchema = new Schema({
+  name: {type: String, required = true},
+  file: {type: String, required = true},
+  owner_id: {type: Number, required = true},
```

Comment:
Should be a foreign key i.e.
{type: Schema.Types.ObjectId, ref: 'User'}

Reply:
added in foreign key